app/helpers/Intersect.js

```diff
@@ -301,18 +311,29 @@ async function asyncCheckIntersect() {
       try {
         let responseJson = await retrieveUrlAsJson(authority.url);
 
+        // get the saved set of locations for the user, normalize and sort
+        let locationArray = normalizeAndSortLocations(
+          await getSavedLocationArray(),
+        );
+
         // Update the news array with the info from the authority
         name_news.push({
           name: responseJson.authority_name,
           news_url: responseJson.info_website,
         });
 
         // intersect the users location with the locations from the authority
-        let tempDayBin = intersectSetIntoBins(
+        let { bins, newIntersections } = intersectSetIntoBins(
           locationArray,
           normalizeAndSortLocations(responseJson.concern_points),
         );
 
+        // Updates crossedPaths field within location data if a user has crossed paths with a new infected case
```

Comment:
Another point here is that someone is working on the geolocation store to make it encrypted. That means writing back will change. So, I think it's a good idea to keep this new structure separate so that those incoming changes don't invalidate this work. Thanks again for adjusting to our needs!